### PR TITLE
Build against Elasticsearch 7.7.1

### PR DESCRIPTION
**Pull request by sruti1312**
*Issue #, if available:*

*Description of changes:*
Build against Elasticsearch 7.7.1
Release 1.8.1

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -56,6 +56,7 @@ test {
 }
 
 licenseHeaders.enabled = false
+validateNebulaPom.enabled = false
```
Comment (ricardolstephen) [inline build.gradle:59]: Why is this change needed?

Comment (sruti1312) [reply build.gradle:59]: ES 7.7.1 requires Gradle 6.4.1 but the latest Nebula only supports Gradle 6.4
Currently, stating validateNebulaPom.enabled = false in the build.gradle script to avoid build failure. 
Need to update this if there is better solution.
